client/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why do we have two license files instead of a single file in the repository root?